Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:2
+
+WORKDIR /s3-tests
+
+RUN apt-get install -y libevent-dev libxml2-dev libxslt-dev zlib1g-dev
+COPY . .
+
+# slightly old version of setuptools; newer fails w/ requests 0.14.0
```

Comment:
What does it request?

Reply:
It may have been an old and redundant need to pin the dependency, it was copied directly from the `bootstrap` script. Turns out it's not even needed any longer, so I just removed it.